motor/discovery/azure/resolver.go

```diff
@@ -140,18 +143,15 @@ func (r *Resolver) Resolve(ctx context.Context, root *asset.Asset, tc *providers
 				},
 			}
 			subsAssets[*sub.Sub.SubscriptionID] = subAsset
-			// if there's a root, link the sub to it
```

Comment:
did the related assets stuff get moved somewhere else/gets picked up elsewhere in the file?

Reply:
i removed the links to the roots. right now the root is an empty azure object which brings no value. the assets are already being linked to the sub they belong to. i think we should fix the root to make it more sensible (or just have no root?) and then we can bring the root linking